test/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_14)
 add_test(test_all ${PROJECT_NAME})
 
 file(COPY data DESTINATION . FILES_MATCHING PATTERN "*.ply")
+if(EMSCRIPTEN)
+    set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS 
+        "-s ASSERTIONS=1 -s DEMANGLE_SUPPORT=1 --preload-file data --bind")
```

Comment:
Just for my own information, since I've never used emscripten, what do these options do? Are they all necessary to allow this to build?

Reply:
The first two options are for better diagnostics, or node will just say undefined error or something when it crashes.
The last argument `--preload-file data --bind` is to load the assets into [emscripten filesystem](https://emscripten.org/docs/porting/files/file_systems_overview.html#emscripten-file-system-runtime-environment) without accessing the host file system.